operator/v1/0000_80_csi_snapshot_operator_01_crd.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: csisnapshots.operator.openshift.io
+spec:
+  group: operator.openshift.io
+  names:
+    kind: CSISnapshot
+    plural: csisnapshots
+    singular: csisnapshot
+  preserveUnknownFields: false
+  scope: Cluster
+  subresources:
+    status: {}
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  validation:
+    openAPIV3Schema:
+      description: KubeAPIServer provides information to configure an operator to
+        manage kube-apiserver.
+      type: object
```

Comment:
description needs fixing.

Reply:
This has been corrected for the current CSISnapshot object. If the Kind changes, then I'll update this again.